pkg/custom-plugins/component.go

```diff
@@ -162,6 +154,19 @@ func (c *component) Check() components.CheckResult {
 		}
 	}
 
+	if cr.err != nil {
+		cr.health = apiv1.HealthStateTypeUnhealthy
+		cr.reason = fmt.Sprintf("error executing state plugin -- %s (output: %s)", cr.err, string(cr.out))
```

Comment:
don't populate the reason with the cr.out since the script may print lots of text.

Reply:
fixed